### PR TITLE
Singh test

### PR DESCRIPTION
**Pull request by iluxa-com**

### CODE REVIEW

**Comment by iluxa-com** [comment]
up - GH review comment

**Comment by iluxa-c0m** [comment]
gh-> up

File: README.MD [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ;)
 ::/:
 
+
 [![Build Status](https://travis-ci.org/domnikl/DesignPatternsPHP.svg?branch=master)](https://travis-ci.org/domnikl/DesignPatternsPHP)
```
Comment (iluxa-com) [inline README.MD:9]: up -> gh inline comment

File: README.MD [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ;)
```
Comment (iluxa-com) [inline README.MD:5]: |`line 2`|
|---|
up-> gh out of the diff

**Comment by iluxa-com** [comment]
|`README.MD:48`|
|---|
48 line

File: my.test [hunk 3]
```diff
@@ -6,5 +6,6 @@ sadf111
 sdfa
 web33441111
```
Comment (iluxa-com) [inline my.test:7]: gh -> up 7 line

**Review by iluxa-com — COMMENTED** [review]
review finished

**Pull request merged**